src/main/java/io/confluent/idesidecar/restapi/resources/DirectQueryResource.java

```diff
@@ -44,9 +44,8 @@ public Uni<List<DirectConnection>> getDirectConnections() {
    */
   @Query("directConnectionById")
   @Description("Get direct connection by ID")
-  @NonNull
-  public DirectConnection getDirectConnectionByID(String connectionID) throws Exception {
-    return direct.getDirectConnectionByID(connectionID);
+  public Uni<DirectConnection> getDirectConnectionByID(@NonNull String id) {
```

Comment:
Adding `@NonNull` to the parameter `id` because we don't want to accept `null` values.